core/src/test/java/org/kohsuke/stapler/export/JSONDataWriterTest.java

```diff
@@ -74,4 +78,48 @@ public Supers(Super... elements) {
         }
     }
 
+    @Test
+    public void JSONObjectWithoutNullValue() throws IOException {
+        String json = "{'key':'value', 'optional':'present'}";
+
+        StringWriter w = new StringWriter();
+        ModelWithJsonField jsonModel = new ModelWithJsonField(json);
+
+        DataWriter writer = JSON.createDataWriter(jsonModel,w);
+        Model<ModelWithJsonField> model = new ModelBuilder().get(ModelWithJsonField.class);
+        model.writeTo(jsonModel, writer);
+
+        assertTrue("Generated JSON :"+w.toString()+ "is not a correct representation of :"+json,
+                   w.toString().contains("present"));
+    }
+
+    @Test
```

Comment:
To make mergeable, either offer a fix or
```suggestion
    @Ignore
    @Test
```

Reply:
I tried to reproduce the issue, so it's normal that the test is failing. The goal is to fix the library to make it work fine with null value.

Reply:
Right, as I said, for this PR to be taken out of draft status and potentially merged, it would need to _either_

* add an `@Ignore` to the test, to document a known limitation; or
* fix `src/main/`